src/avfoundation.cs

```diff
@@ -1261,6 +1504,9 @@ interface AVAudioPlayerDelegate {
 #endif
 	}
 
+	[Watch (4,0), TV (11,0), Mac (10,13), iOS (11,0)]
+	delegate void AVAudioPlayerNodeCompletionHandler (AVAudioPlayerNodeCompletionCallbackType type);
```

Comment:
does it exists in ObjC ?
A single parameter delegate can be turned into an `Action<T>` like
`Action<AVAudioPlayerNodeCompletionCallbackType>`
as it remains clear what it is doing (the `type` parameter name does not help in this case)